doc/source/whatsnew/v1.3.0.rst

```diff
@@ -274,6 +274,7 @@ Other enhancements
 - Add keyword ``dropna`` to :meth:`DataFrame.value_counts` to allow counting rows that include ``NA`` values (:issue:`41325`)
 - :meth:`Series.replace` will now cast results to ``PeriodDtype`` where possible instead of ``object`` dtype (:issue:`41526`)
 - Improved error message in ``corr`` and ``cov`` methods on :class:`.Rolling`, :class:`.Expanding`, and :class:`.ExponentialMovingWindow` when ``other`` is not a :class:`DataFrame` or :class:`Series` (:issue:`41741`)
+- :class:`TextReader` and everything that uses the class supports defaultdict as dtype
```

Comment:
move to 1.4; TextReader is not public, rather just say read_csv

Reply:
> move to 1.4

Who knows, may be the PR will not be closed and by 1.5 with such a PR lifecycle. So I suppose it must be done in the end in order to avoid more conflicts.

> TextReader is not public, rather just say read_csv

Does TextReader work only for read_csv?